source/DSCResources/xSccmPreReqs/xSccmPreReqs.schema.psm1

```diff
@@ -193,7 +193,7 @@ Configuration xSCCMPreReqs
     )
 
     Import-DscResource -ModuleName PSDesiredStateConfiguration
-    Import-DscResource -ModuleName NetworkingDsc -ModuleVersion 7.4.0.0
+    Import-DscResource -ModuleName NetworkingDsc
```

Comment:
-ModuleVersion 8.2.0    should be added to this line.

Reply:
I respectfully disagree. We are resolving module dependencies during builds and update the module path, so the module version that is referenced in `RequiredModules.psd1` is be the one that is being used to compile the configurations. Using a specific module version will invariably lead to the exact same issue that I would like to see fixed with this PR in the future. Unless there is a considerably faster release cycle, hard-coded module versions seem like a bad choice to me personally.

Ultimately that decision lies with you guys. If hard-coded module versions are a requirement, I would rather close this PR as it will be easier for my customers to just fork the code and release their own version internally. I would have thought that other users had the same problem though, but if it is only me and my customers then I guess that's fine.